api/app/reviews/tests.py

```diff
@@ -1518,17 +1514,9 @@ def static_seed_data(self):
         db.session.add_all(review_forms)
         db.session.commit()
 
-        self.test_response = Response(  # Nominating other
-            1, self.first_user.id)
-        self.test_response.submit()
+        self.test_response = self.add_response(1, self.first_user.id, language='en')

```

Comment:
@avishkar58 
I think I'm not clicking on something, but this and line `1519` are returning a `400` in the tests (not 200). 

Do you have any hints for me by any chance, please? :)

Reply:
Hey Siobhan, can you update which line/test you're referring to here?

Reply:
Oh nevermind, I see where it's coming from now. So status code 400 means "bad request" and this would be thrown by the request parser if the request was either missing a required parameter or you passed data with the wrong datatype to a parameter (eg it was expecting an integer but you passed a string). In this case, you can see the Review API's request parser is defined [here](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/api/app/reviews/mixins.py#L4) - and it has 2 required parameters, `event_id` and `language`. In your tests, for example in line 1533, you're only passing the event_id which is why you're getting the 400. Hope that makes sense. Have a look [here](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/api/app/reviews/tests.py#L195) for an example that passes both the event_id and the language.